app/assets/javascripts/hyrax/save_work/validation_service.es6

```diff
@@ -0,0 +1,63 @@
+export default class ValidationService {
+  /**
+   * @param {Form} element
+   */
+  constructor(form, new_work) {
+    this.form = form
+    this.form.on('submit', (event) => { this.checkValidationService(event, new_work) })
+    this.setupAlert()
+    this.readOnlyAccession(new_work)
+  }
+
+  /**
+   * check the validation service before submit
+   *
+   */
+  checkValidationService(event, new_work) {
+      event.preventDefault()
+      this.closeAlert()
+      console.log('Checking validation service...');
```

Comment:
I'd remove additional console logs, if they're not being used for specific error message helpers.  General convention is to avoid them in production code, and some linters will complain about this.